### ISSUE TITLE
isGroupTransactionPart failed because of missing value brq_transactions

### ISSUE THREAD

**Issue by JoostWan** [buckaroo-it/Magento2#42]
When Buckaroo push the data after a successful checkout there is a error in the Magento logs; 

`[2020-07-30 07:06:36] main.CRITICAL: Report ID: webapi-5f22717c3684d; Message: Notice: Undefined index: brq_transactions in /home/xxx/releases/20200729072923/vendor/buckaroo/magento2/Model/Push.php on line 1193 {"exception":"[object] (Exception(code: 0): Report ID: webapi-5f22717c3684d; Message: Notice: Undefined index: brq_transactions in /home/xxx/releases/20200729072923/vendor/buckaroo/magento2/Model/Push.php on line 1193 at /home/xxx/releases/20200729072923/vendor/magento/framework/Webapi/ErrorProcessor.php:208, Exception(code: 0): Notice: Undefined index: brq_transactions in /home/xxx/releases/20200729072923/vendor/buckaroo/magento2/Model/Push.php on line 1193 at /home/xxx/releases/20200729072923/vendor/magento/framework/App/ErrorHandler.php:61)"} []`

There is a check if a transactions is a grouptransactionPart in this code

```
    private function isGroupTransactionPart()
    {
        return $this->groupTransaction->getGroupTransactionByTrxId($this->originalPostData['brq_transactions']);
    }
```

But in our case Buckaroo push doesn't contain brq_transactions and it will generate a PHP error and the response is not processed. 

This is the response from buckaroo 

```
[30-Jul-2020 07:12:20 UTC] Array
(
    [ADD_initiated_by_magento] => 1
    [ADD_service_action_from_magento] => reserve
    [brq_datarequest] => *****************
    [brq_datarequest_] => *****************
    [brq_invoicenumber] => *****************
    [brq_primary_service] => KlarnaKp
    [brq_SERVICE_klarnakp_ReservationNumber] => *****************
    [brq_statuscode] => 190
    [brq_statuscode_detail] => S990
    [brq_statusmessage] => The request was successful.
    [brq_test] => true
    [brq_timestamp] => 2020-07-30 09:12:20
    [brq_websitekey] => *****************
    [brq_signature] => *****************
)

```

**Comment by vladislav-padalka-hys** [comment]
Hi @JoostWan !
Thanks for you report !
This error has been fixed ~2 weeks ago, but as for know it's available in master branch only
We expect to release new public version soon which will include that fix as well.

**Comment by JoostWan** [comment]
When there will be a composer release with this issue fixed?

**Comment by vladislav-padalka-hys** [comment]
we expect RC to be published today or tomorrow

**Comment by Buckaroo-Rens** [comment]
Issue was fixed in 1.25.0-RC1
